be/src/exec/schema_scanner/schema_loads_scanner.cpp

```diff
@@ -0,0 +1,286 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "exec/schema_scanner/schema_loads_scanner.h"
+
+#include "exec/schema_scanner/schema_helper.h"
+#include "http/http_client.h"
+#include "runtime/runtime_state.h"
+#include "runtime/string_value.h"
+
+namespace starrocks {
+
+SchemaScanner::ColumnDesc SchemaLoadsScanner::_s_tbls_columns[] = {
```

Comment:
Where to persist the message?

Reply:
There is no difference in meta persist with previous implementation, this PR just adds the SQL interface obtained through information_schema